.travis.yml

```diff
@@ -19,6 +19,10 @@ python:
   - '2.7'
   - '3.6'
 
+before_install:
+  # Support processing of core dump
+  - sudo apt-get install -y gdb
```

Comment:
As discussed in person, we can probably install gdb in the after failure so that it only gets installed when we actually need it